README.md

```diff
@@ -9,7 +9,7 @@
 - [Prerequisites](#prerequisites)
 - [Get Started](#get-started)
 - [Options](#options)
-  - [Component Versions](#component-versions)
+  - [Component Versions](#component-versions-and-exposed-ports)
   - [All Options](#all-options)
 - [Docker Building Blocks](#docker-building-blocks)
   - [Aquarius](#aquarius)
```

Comment:
This will run the current default versions of Aquarius, Provider, and Ganache with our contracts deployed to it.